### ISSUE TITLE
Alpha channel for text (font)

### ISSUE THREAD

**Issue by felixbecker** [jgraph/drawio#2472]
* [ x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I would like to have alpha channels on fonts to adjust transparency on text. For example if you want to watermark something or have a transparent text.
As far as I know is it possible to have an alpha on shapes but not on fonts.

**Describe the solution you'd like**

Basically use css syntax either hey code with the last two digets for alpha (#9999997d grey with 50% alpha) or rgba() values 

**Describe alternatives you've considered**
Maybe there is already a way an I just don't know it.

**Additional context**

**Comment by alderg** [comment]
Text opacity should work:
<img width="243" alt="Screenshot 2021-12-06 at 05 59 41" src="https://user-images.githubusercontent.com/921200/144789888-7bb1c25b-a2d0-4250-a614-e26b8d4a6774.png">
